src/Ombi.Notifications/Agents/DiscordNotification.cs

```diff
@@ -21,12 +22,34 @@ public class DiscordNotification : BaseNotification<DiscordNotificationSettings>
         public DiscordNotification(IDiscordApi api, ISettingsService<DiscordNotificationSettings> sn,
                                    ILogger<DiscordNotification> log, INotificationTemplatesRepository r,
                                    IMovieRequestRepository m, ITvRequestRepository t, ISettingsService<CustomizationSettings> s)
-            : base(sn, r, m, t,s,log)
+            : base(sn, r, m, t, s, log)
         {
             Api = api;
             Logger = log;
+            ShowCompactEmbed = new Dictionary<NotificationType, bool>();
+            // Temporary defaults
+            ShowCompactEmbed.Add(NotificationType.NewRequest, true);
+            ShowCompactEmbed.Add(NotificationType.RequestApproved, true);
+            ShowCompactEmbed.Add(NotificationType.RequestDeclined, true);
+
+            MentionAlias = new Dictionary<NotificationType, bool>();
+            // Temporary defaults
+            MentionAlias.Add(NotificationType.RequestAvailable, true);
+            MentionAlias.Add(NotificationType.RequestApproved, true);
+            MentionAlias.Add(NotificationType.RequestDeclined, true);
         }
 
+        // constants I needed but could not find
+        public const string IMDB_BASE_URL = "http://www.imdb.com/title/";
+        public const string TVDB_BASE_URL = "https://www.thetvdb.com/?tab=series&id=";
+
+        // Whether or not to show a compact embed notification (thumbnail + no description)
+        public Dictionary<NotificationType, bool> ShowCompactEmbed;
+        // Whether or not to post the alias to discord (for each notification type) which trigger an @mention if alias is set up as <@id>.  
+        public Dictionary<NotificationType, bool> MentionAlias;
+        // Whether to use alias or username when referencing a user
+        public bool UsingAliasAsMention = true;
```

Comment:
What's the difference between `UsingAliasAsMention` and `MentionAlias`?

Reply:
Thinking about it i'm not sure if this option is needed since the in the body the user could just enter `@<{Alias}>`. What do you think?